client/core/types.go

```diff
@@ -498,15 +499,15 @@ type PendingFeeState struct {
 
 // Exchange represents a single DEX with any number of markets.
 type Exchange struct {
-	Host       string                `json:"host"`
-	AcctID     string                `json:"acctID"`
-	Markets    map[string]*Market    `json:"markets"`
-	Assets     map[uint32]*dex.Asset `json:"assets"`
-	Connected  bool                  `json:"connected"`
-	Fee        *FeeAsset             `json:"feeAsset"` // DEPRECATED. DCR.
-	RegFees    map[string]*FeeAsset  `json:"regFees"`
-	PendingFee *PendingFeeState      `json:"pendingFee,omitempty"`
-	CandleDurs []string              `json:"candleDurs"`
+	Host             string                 `json:"host"`
+	AcctID           string                 `json:"acctID"`
+	Markets          map[string]*Market     `json:"markets"`
+	Assets           map[uint32]*dex.Asset  `json:"assets"`
+	ConnectionStatus comms.ConnectionStatus `json:"connectionStatus"`
```

Comment:
Ok because we are still at v0.x but note that when we reach v1, we're making a promise to Go consumers not to break the API.  So doing something like this in the future would cause the module to become `core/v2`. 

However, if we left `Connected bool`, it would not be a breaking change even if it appears redundant with `ConnectionStatus`.

Just pointing it out so we can get used to stabilizing the API.